### ISSUE TITLE
Dropout rate

### ISSUE THREAD

**Issue by ahmdtaha** [pudae/tensorflow-densenet#20]
In densenet.py Line: 48

shouldn't it be ??
    if dropout_rate:
    >>>>>net = tf.nn.dropout(net,**keep_prob=dropout_rate**)
